src/plugins/home/public/application/components/tutorial_directory.js

```diff
@@ -66,45 +66,48 @@ class TutorialDirectoryUi extends React.Component {
   constructor(props) {
     super(props);
 
+    // temporarily set tabs to empty due to no available data instructions
```

Comment:
NIT:
I have be appending `// TODO: [RENAMEME]` for stuff we want but do not currently have a replacement for.
Makes it easier to find in the future.

Reply:
I will formalize this comment. Thanks bro